### ISSUE TITLE
Powershell breaks cursor keys by setting DECCKM

### ISSUE THREAD

**Issue by gaborbernat** [wez/wezterm#510]
## Describe the bug

When using [Github CLI](https://cli.github.com/) - written in Go - the tool sometimes asks you questions in an interactive format. You can use up/down arrow to select an option. The easiest such is ``gh auth`` post fresh installation. When you press the arrows the command fails with ``Unexpected Escape Sequence: ['\x1b' 'O']``. Once I set ``send_composed_key_when_alt_is_pressed = false`` the failure no longer happens. I've only tried on macOS.

## Environment (please complete the following information):

 - OS: macOS
 - Version: ``wezterm 20210203-095643-70a364eb-133-g48f5fc``

## To Reproduce

* install ``github-cli``
* run ``gh auth``
* press up/down arrows

## Configuration

https://github.com/gaborbernat/dotfiles/blob/4cb9841cc9c29551cde5a385aa580f6064fc119a/dot_config/wezterm/wezterm.lua#L1
https://github.com/gaborbernat/dotfiles/blob/4cb9841cc9c29551cde5a385aa580f6064fc119a/dot_config/wezterm/macOS.lua#L1

**Comment by wez** [comment]
I can't reproduce this; I suspect it may be related to your active keyboard layout; would you mind sharing what that is set to?

**Comment by gaborbernat** [comment]
My keyboard layout is ``British``.

**Comment by wez** [comment]
We talked through this in the matrix channel, the gist of it is:

* `\x1bO` is the encoding used to represent cursor keys when [DECCKM](https://vt100.net/docs/vt510-rm/DECCKM.html) is enabled.  The default is disabled and cursor up is represented as the ANSI `CSI A`.  When `DECCKM` is enabled `SS3 A` is used instead.
* We figured out that powershell is setting `DECCKM` for its line editing, but not clearing it when launching programs
* It's probably also a bug in the GH CLI that it doesn't recognize application key encoding.
* The same behavior occurs in Terminal.app
